src/context/AuthContext.js

```diff
@@ -26,6 +27,21 @@ export function AuthProvider({ children }) {
       : browserSessionPersistence;
     await setPersistence(auth, persistenceMethod);
     await signInWithEmailAndPassword(auth, email, password);
+
+    auth.currentUser
+      .getIdToken(/* forceRefresh */ true)
+      .then(function (idToken) {
```

Comment:
cool, gonna test it now.
btw, why not using `async & await` instead of `then` ?

Reply:
Oh, I can def change this to async/await!